scripts/release/utils/workspace.ts

```diff
@@ -27,8 +27,10 @@ const writeFile = promisify(_writeFile);
 
 const {
   workspaces: rawWorkspaces
-}: { workspaces: string[] } = require(`${root}/package.json`);
-const workspaces = rawWorkspaces.map(workspace => `${root}/${workspace}`);
+}: { workspaces: { packages: string[] } } = require(`${root}/package.json`);
```

Comment:
These changes are from #8920 . I pulled the branch from that PR into the API change (#1) in anticipation of using the type package. Now that we're not using that package, we should revert those changes.